test/views.ts

```diff
@@ -26,8 +22,8 @@ import crossReference from '../src/views/cross_reference_editable'
 import { EditableBlockProps } from '../src/views/editable_block'
 import equation from '../src/views/equation_editable'
 import equationElement from '../src/views/equation_element_editable'
-import Figure, { FigureProps } from '../src/views/FigureComponent'
-import FigureElement from '../src/views/FigureElement'
+import figure from '../src/views/figure'
```

Comment:
fyi these are not the ones that we use in the current setup. See src/configs/lean-workflow/editor-views-lw.ts